README.md

```diff
@@ -71,7 +71,43 @@ Required input:
 </p>    
 </div>
 
-### Resources
+## <div align="center">High level features and techniques</div>
+
+MCT offers a range of powerful features to optimize models for efficient edge deployment. These supported features include:
+
+### Quantization Core Features
+
+🏆 **Mixed-precision search** [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mixed_precision_ptq.ipynb). Assigning optimal quantization bit-width per layer (for weights/activations) 
+
+📈 **Graph optimizations**. Transforming the model to be best fitted for quantization process.
+
+🔎 **Quantization parameter search**. Minimizing expected quantization-noise during thresholds search using methods such as MSE, No-Clipping and MAE.
+
+🧮 **Advanced quantization algorithms**. To prevent a performance degradation some algorithms are applied, such as Shift negative correction, Outliers filtering and clustering.
```

Comment:
I think this sentence needs rephrasing, first of all, the mentioned features are not applead by default (so maybe say that they are "available"). Second, I would remove the "prevent performance degradation", and change this to something that sounds positive ("we provide additional feature that allows to enhance the quantization performance for advanced cases..." something like that).
Finally, you can add links to tutorials here as well

Reply:
Fixed